test/integration/proposal-checks/IntegrationTestBorrowSDAICollateral.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.13;
+import {GovernorCountingSimple} from "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol";
+import {Governor, IGovernor} from "@openzeppelin/contracts/governance/Governor.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+
+import "@forge-std/Test.sol";
+
+import {Core} from "@src/core/Core.sol";
+import {CoreRoles} from "@src/core/CoreRoles.sol";
+import {MockERC20} from "@test/mock/MockERC20.sol";
+import {GuildToken} from "@src/tokens/GuildToken.sol";
+import {LendingTerm} from "@src/loan/LendingTerm.sol";
+import {VoltGovernor} from "@src/governance/VoltGovernor.sol";
+import {NameLib as strings} from "@src/utils/NameLib.sol";
+import {CoreRoles as roles} from "@src/core/CoreRoles.sol";
+import {PostProposalCheck} from "@test/integration/proposal-checks/PostProposalCheck.sol";
+import {LendingTermOnboarding} from "@src/governance/LendingTermOnboarding.sol";
+import {LendingTermOffboarding} from "@src/governance/LendingTermOffboarding.sol";
+import {ProtocolConstants as constants} from "@src/utils/ProtocolConstants.sol";
+
+contract IntegrationTestBorrowSDAICollateral is PostProposalCheck {
+    function setUp() public override {
+        super.setUp();
+    }
+
```

Comment:
Vote, veto, borrow, repay, partial repay, add collateral, remove collateral

Reply:
setup 3 users, one that is rebasing, one that is not rebasing, one that is going to borrow, create bad debt, run through different scenarios and see what happens to first 2 users.

borrow cap changes when total supply of CREDIT changes